.github/workflows/awsfulltest.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: nf-core/tower-action@v2
+        uses: nf-core/tower-action@v3
```

Comment:
In this file `pipeline: ...` and `revision: ....` should be removed. The change was introduced in the latest template update

Reply:
Somehow I missed these lines